docs/eventing/samples/ping-source/README.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "PingSource example"
+linkTitle: "PingSource"
+weight: 10
+type: "docs"
+---
+
+PingSource example shows how to configure PingSource as event source for
+functions.
+
+## Before you begin
+
+1. Setup [Knative Serving](../../../serving).
+1. Setup [Knative Eventing](../../../eventing).
+
+## Create a Knative Service
+
+In order to verify `PingSource` is working, we will create a simple Knative
+Service that dumps incoming messages to its log.
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+spec:
+  template:
+    spec:
+      containers:
+        - image: gcr.io/knative-releases/github.com/knative/eventing-sources/cmd/event_display
+```
+
+Use following command to create the service from `service.yaml`:
+
+```shell
+kubectl apply --filename service.yaml
+```
+
+## Create a PingSource
+
+For each set of ping events you want to request, you need to create an Event
+Source in the same namespace as the destination. If you need a different
+ServiceAccount to create the Deployment, modify the entry accordingly in the
+yaml.
+
+```yaml
+apiVersion: sources.knative.dev/v1alpha1
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/2 * * * *"
+  data: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+```
+
+Use following command to create the event source from `ping-source.yaml`:
+
+```shell
+kubectl apply --filename ping-source.yaml
+```
+
+## Verify
+
+We will verify that the message was sent to the Knative eventing system by
+looking at message dumper logs.
+
+```shell
+kubectl logs -l serving.knative.dev/service=event-display -c user-container --since=10m
+```
+
+You should see log lines showing the request headers and body from the source:
+
+```
+☁️  cloudevents.Event
+Validation: valid
+Context Attributes,
+  specversion: 1.0
+  type: dev.knative.sources.ping
+  source: /apis/v1/namespaces/default/cronjobsources/test-cronjob-source
+  id: d8e761eb-30c7-49a3-a421-cd5895239f2d
+  time: 2019-12-04T14:24:00.000702251Z
+  datacontenttype: application/json
+Data,
+  {
+    "message": "Hello world!"
+  }
+```
+
+You can also use [`kail`](https://github.com/boz/kail) instead of `kubectl logs`
+to tail the logs of the subscriber.
+
+```shell
+kail -l serving.knative.dev/service=event-display -c user-container --since=10m
+```
+
+## Cleanup
+
+You can delete the PingSource by entering the following command:
```

Comment:
What exactly are you deleting here? Is it the event source? Or the YAML file? Can we add a noun to the sentence? i.e. "You can delete the Pingsource [NOUN] by entering the following command..."

Reply:
My guess is the event source is created the same way as a service by applying a yaml file, so deleting the yaml file is deleting the source effectively?